GameProject/Assets/TurnBasedAssets/Scripts/Controllers/Controller.cs

```diff
@@ -0,0 +1,13 @@
+﻿using TurnBasedAssets.Scripts.Interface;
+using UnityEngine;
+
+namespace TurnBasedAssets.Scripts.Controllers
+{
+    public class Controller : MonoBehaviour
```

Comment:
Empty type? Any reason for this being here?

Reply:
Currently any controllers inherit from this type, including an object controller. The function is to have them all handle re positioning themselves automatically to the correct axis, which will also extend to all enemy AI. Having them inherit from this type allowed me to have them all send themselves as the argument and it fulfilling a single argument parameter.

Reply:
Made this class no longer empty